session.go

```diff
@@ -327,7 +334,7 @@ func (s *session) Retry() error {
 			return errors.Trace(err)
 		}
 		retryCnt++
-		if (s.maxRetryCnt != unlimitedRetryCnt) && (retryCnt >= s.maxRetryCnt) {
+		if !s.unlimitedRetryCount && (retryCnt >= maxCnt) {
 			return errors.Trace(err)
 		}
 		kv.BackOff(retryCnt)
```

Comment:
If a large txn failed, the system may need more time to recover. So should we wait a longer time? @disksing What's your opinion？